client/components/domains/registrant-extra-info/docs/example.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+const countriesList = require( 'lib/countries-list' ).forDomainRegistrations(),
+	Card = require( 'components/card' ),
+	ExtraInfoFrForm = require( 'components/domains/registrant-extra-info/fr-form' ).default;
+
+class ExtraInfoFrFormExample extends PureComponent {
+	static displayName = 'DomainsFrExtraForm'
+
+	constructor( props ) {
+		super( props );
+
+		this.state = {
```

Comment:
Unless I'm missing something - it's not a weird syntax :) It's so normal that it's weird 😀 You're doing the same thing you're doing in the constructor - assigning an instance variable `state` some value.